types/intset.go

```diff
@@ -1,9 +1,35 @@
 package types
 
 import (
+	"fmt"
 	"github.com/Workiva/go-datastructures/bitarray"
+	cbor "github.com/ipfs/go-ipld-cbor"
+	"github.com/polydawn/refmt/obj/atlas"
 )
 
+func init() {
+	cbor.RegisterCborType(intSetAtlasEntry)
+}
+
+var intSetAtlasEntry = atlas.BuildEntry(IntSet{}).Transform().
+	TransformMarshal(atlas.MakeMarshalTransformFunc(
+		func(is IntSet) ([]byte, error) {
+			// TODO #2889: we should be using RLE+ to serialize intsets when it lands,
```

Comment:
cc @Ingar when taking #2889 these calls should be updated with `is.ToRLE()` and `IntSetFromRLE()` (or whatever interface you go with)